packages/manager/modules/anthos/src/anthos-tenants.service.js

```diff
@@ -8,28 +8,22 @@ export default class AnthosTenantsService {
     this.ovhDocUrl = ovhDocUrl;
   }
 
+  fetch(endPoint) {
```

Comment:
Not sure to get the point of this abstraction

Reply:
To avoid for each ```$http.get``` to rewrite ```then(({data}) => data)```.

Reply:
I think it unnecessarily complicates things... Its ok to have `then(({data}) => data)` in each call, according to me

Reply:
For me, it will reduce the weight of app.
i think it's a good practice.